src/RiakClient/Models/MapReduce/KeyFilters/And.cs

```diff
@@ -13,64 +15,90 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-
-using Newtonsoft.Json;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
+// </copyright>
 
 namespace RiakClient.Models.MapReduce.KeyFilters
 {
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using Newtonsoft.Json;
+
     /// <summary>
     /// Joins two or more key-filter operations with a logical AND operation.
     /// </summary>
     internal class And : IRiakKeyFilterToken
     {
-        private readonly Tuple<string, List<IRiakKeyFilterToken>, List<IRiakKeyFilterToken>> _kfDefinition;
+        /// <summary>
+        ///  Collection of key filter definitions.
+        /// </summary>
+        private readonly Tuple<string, List<IRiakKeyFilterToken>, List<IRiakKeyFilterToken>> keyFilterDefinition;
```

Comment:
I also like avoiding pseudo hungarian notation, I think Uncle Bob warns of it in "Clean Code" too.  Most of the time we're using intellisense so the extra characters aren't an issue.  

I would vote for the CoreFX version of underscore usage over `this` here, it's mostly a personal preference though.  Using a `_` gives the reader another clue to the scoping of the variable, which is nice in places where you don't have "Find Declaration..." (i.e. github reviews).  You can omit a `this.` however if you don't have stylecop error out things in the build.  6 one way, half a dozen the other.